network/manager.go

```diff
@@ -253,3 +253,7 @@ func (mgr *Manager) AddPeer(conn net.Conn, nonce []byte) error {
 
 	return nil
 }
+
+func createTCPListener(addr *net.TCPAddr) (Listener, error) {
```

Comment:
I would also remove the reference to TCP everywhere. It's not necessary, because we only use TCP and not any kind of other listeners or whatever.

Reply:
Removed this func.